src/Ide/Cradle.hs

```diff
@@ -517,6 +519,7 @@ cabalHelperCradle file = do
                                         realPackage
                                         normalisedPackageLocation
                                         fp
+                                    , runGhcCmd = \_ -> pure CradleNone
```

Comment:
We should switch on the project type from c-h and also call `cabal/stack exec ghc`

Reply:
@bubba We dont want to use cabal-helper anymore, afaict. The PR #88 would remove this cradle entirely in a follow up PR.
However, if we want to adapt c-h to use the new API, then https://hackage.haskell.org/package/cabal-helper-1.1.0.0/docs/Distribution-Helper.html#v:qePrograms would be probably helpful. (and trivial to implement, afaict)

Reply:
Yeah I was thinking if we remove it then this might be a bit of waste. Nothing in hls in master is actually calling the `getRuntimeLibDir`/`getRuntimeGhcVersion` functions currently so I don't think anything will break

Reply:
Fixed it anyways (untested, though)